mods/tuxemon/maps/taba_town.tmx

```diff
@@ -89,7 +89,7 @@
   <object id="9" type="collision" x="640" y="800" width="96" height="32"/>
   <object id="10" type="collision" x="640" y="832" width="64" height="16"/>
   <object id="11" type="collision" x="720" y="832" width="16" height="16"/>
-  <object id="12" type="collision" x="736" y="640" width="96" height="320"/>
+  <object id="12" type="collision" x="736" y="640" width="96" height="207.667"/>
```

Comment:
float values should be avoided.  please make sure all events/collisions fit on the tile grid and are whole numbers.